components/src/lib/AlertModal/index.tsx

```
@@ -0,0 +1,42 @@
+      title={title}
+      size={size}
+      hasCloseButton={false}
+      modalRef={modalRef}
+      content={<S.AlertText>{alertText}</S.AlertText>}
+      buttonElements={[
+        <Button
+          autoFocus
+          variant="primary"
+          type="button"
+          onClick={onRequestClose}
+        >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)